### PR TITLE
Check Antagonists player counting now excludes CTF and living players on central command.

### PR DESCRIPTION
**Pull request by silicons**
Zlevel 2 players get counted separately.

### CODE REVIEW

**Comment by Kromgar** [comment]
Wew

File: code/modules/admin/player_panel.dm [hunk 1]
```diff
@@ -348,12 +348,22 @@
 		var/living_players_antagonist = 0
 		var/brains = 0
 		var/other_players = 0
+		var/living_skipped = 0
+		var/ctf_mobs = 0
 		for(var/mob/M in mob_list)
+			if(ishuman(M))
+				var/mob/living/carbon/human/H = M
+				if(istype(H.wear_suit, /obj/item/clothing/suit/space/hardsuit/shielded/ctf))
```
Comment (lzimann) [inline code/modules/admin/player_panel.dm]: This is a really ugly check. If you are already skipping mobs on z2 you don't need to check for ctf mobs.

Comment (kevinz000) [inline code/modules/admin/player_panel.dm]: K

File: code/modules/admin/player_panel.dm [hunk 2]
```diff
@@ -348,12 +348,22 @@
...
+				var/mob/living/carbon/human/H = M
+				if(istype(H.wear_suit, /obj/item/clothing/suit/space/hardsuit/shielded/ctf))
+					ctf_mobs++
+					continue
 			if(M.ckey)
 				if(isnewplayer(M))
 					lobby_players++
 					continue
 				else if(M.stat != DEAD && M.mind && !isbrain(M))
+					if(M.z == 2)
```
Comment (lzimann) [inline code/modules/admin/player_panel.dm]: use ZLEVEL_CENTCOM instead of 2

**Pull request merged**